app/src/main/java/de/westnordost/streetcomplete/quests/SharedFilters.kt

```diff
@@ -0,0 +1,12 @@
+package de.westnordost.streetcomplete.quests
```

Comment:
Such things should go into the osm package, e.g. into MaxspeedZone.kt or Maxspeed.kt.

Reply:
I placed it there initially but somehow convinced myself it would be better in quests. But you're right. This could be used by other stuff as well, e.g. overlays.